crates/biome_service/src/file_handlers/grit.rs

```diff
@@ -16,8 +16,32 @@ use super::{
     FormatterCapabilities, ParseResult, ParserCapabilities, SearchCapabilities,
 };
 
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+pub struct GritFormatterSettings {
+    pub line_ending: Option<LineEnding>,
+    pub line_width: Option<LineWidth>,
+    pub indent_width: Option<IndentWidth>,
+    pub indent_style: Option<IndentStyle>,
+    pub quote_style: Option<QuoteStyle>,
```

Comment:
Does grit use different quotes?

Reply:
Good call, we shouldn't need this option for Grit.